### PR TITLE
MIP-30 Adjust Form Score Layout

### PR DESCRIPTION
**Pull request by giavano**
This PR should apply the changes on Frm Score Layout.

### CODE REVIEW

File: CIW/Form3.resx [hunk 1]
```diff
@@ -69,30 +69,6 @@
   <metadata name="Score.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
     <value>True</value>
   </metadata>
-  <metadata name="ID.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
-    <value>True</value>
-  </metadata>
-  <metadata name="Date.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
-    <value>True</value>
```
Comment (fjatimora) [inline CIW/Form3.resx]: sample comment

File: CIW/Form3.resx [hunk 2]
```diff
@@ -69,30 +69,6 @@
...
-  </metadata>
-  <metadata name="Score.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
-    <value>True</value>
-  </metadata>
-  <metadata name="dataGridViewTextBoxColumn1.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
-    <value>True</value>
-  </metadata>
-  <metadata name="dataGridViewTextBoxColumn2.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
-    <value>True</value>
```
Comment (fjatimora) [inline CIW/Form3.resx]: sample comment

File: CIW/Form3.resx (hunk 1 above)
Comment (giavano) [reply CIW/Form3.resx]: this comment has been addressed on the last commits

File: CIW/Form3.resx (hunk 2 above)
Comment (giavano) [reply CIW/Form3.resx]: this comment has been addressed on the last commits

**Pull request merged**